### ISSUE TITLE
Doubled notification

### ISSUE THREAD

**Issue by waclaw66** [SchildiChat/SchildiChat-android#90]
**Describe the bug**
When single message is received two same notifications are shown for a little while in a row.

**To Reproduce**
Just receive a message using matrix.org pusher.

**Expected behavior**
Single notification for a message shall be shown only.

**Screenshots**
https://user-images.githubusercontent.com/15554561/135752592-03af6081-2fc8-42f2-8c41-26fa578685d1.mp4

**Smartphone (please complete the following information):**
 - Device: Xiaomi Mi9T
 - OS: Android 11

**Additional context**
 - App version: 1.3.1.sc45-test14
 - Store: Google Play
 - Homeserver: own Synapse instance

**Upstream relevance**
 - Does this issue exist in Element Android?
Yes: https://github.com/vector-im/element-android/issues/4152

**Comment by SpiritCroc** [comment]
I have not seen this on my devices yet. Does this always (for every single message received) happen for you? Have you changed any of the notification settings?
@p1gp1g any idea why this could happen for us, but not for upstream?

**Comment by waclaw66** [comment]
> I have not seen this on my devices yet. Does this always (for every single message received) happen for you? Have you changed any of the notification settings? @p1gp1g any idea why this could happen for us, but not for upstream?

Yes, it does for every message I receive, no matter native matrix or bridged.
I have tried to re-register pusher additionally, but it didn't help.

**Comment by waclaw66** [comment]
I tried to log off and sign in again. The problem resists. Then I've turned on notifications on Element Android, suddenly I receive doubled notifications on Element Android too :D I seems like upstream problem.
Element Android is so buggy. Btw. I have discovered meanwhile that cross signing verification doesn't work between two Element Android devices :[ The new session is unverified for other users, but verified for the user.
